### PR TITLE
Add links to guides for Micronaut OpenAPI

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

File: src/main/docs/guide/convertToAdoc.adoc [hunk 1]
```diff
@@ -19,3 +19,5 @@ Adoc generator based on Apache Freemarker templates, and you can override full t
 |`micronaut.openapi.adoc.output.filename` | Final `adoc` file name. | Default: the same as openapi spec. filename
 |`micronaut.openapi.adoc.templates.*` | Prefix for custom template names. You can override any of the templates that are used in the default implementation to change only part of the resulting document. |
 |===
+
+TIP: See the guide for https://guides.micronaut.io/latest/micronaut-openapi-adoc.html[Generate API Documentation in Asciidoc with the generated OpenAPI specification of your Micronaut application.] to learn more.
```
Comment (altro3) [inline src/main/docs/guide/convertToAdoc.adoc:23]: Asciidoc -> AsciiDoc

File: src/main/docs/guide/openApiGuides.adoc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+See the following list of guides to learn more about working with OpenApi (Swagger) in the Micronaut Framework:
```
Comment (altro3) [inline src/main/docs/guide/openApiGuides.adoc:1]: OpenApi -> OpenAPI

File: src/main/docs/guide/openApiViews/swaggerui.adoc [hunk 3]
```diff
@@ -51,3 +51,5 @@ micronaut:
 ----
 
 With the above configuration in place when you run your application you can access your Swagger documentation at +http://localhost:8080/swagger-ui+.
+
+TIP: See the guide for https://guides.micronaut.io/latest/micronaut-openapi-swagger-ui.html[Visualize with Swagger-UI an OpenAPI specification of your Micronaut application.] to learn more.
```
Comment (altro3) [inline src/main/docs/guide/openApiViews/swaggerui.adoc:55]: Swagger-UI -> Swagger UI

**Review by altro3 — CHANGES_REQUESTED** [review]

File: src/main/docs/guide/convertToAdoc.adoc (hunk 1 above)
Comment (sdelamo) [reply src/main/docs/guide/convertToAdoc.adoc:23]: You can suggest these kind of changes directly from the UI 
CleanShot 2023-12-12 at 09 25 47@2x

Comment (sdelamo) [inline src/main/docs/guide/convertToAdoc.adoc:23]:
```suggestion
TIP: See the guide for https://guides.micronaut.io/latest/micronaut-openapi-adoc.html[Generate API Documentation in AsciiDoc with the generated OpenAPI specification of your Micronaut application.] to learn more.
```

File: src/main/docs/guide/openApiGuides.adoc (hunk 2 above)
Comment (sdelamo) [inline src/main/docs/guide/openApiGuides.adoc:1]:
```suggestion
See the following list of guides to learn more about working with OpenAPI in the Micronaut Framework:
```

File: src/main/docs/guide/openApiViews/swaggerui.adoc (hunk 3 above)
Comment (sdelamo) [inline src/main/docs/guide/openApiViews/swaggerui.adoc:55]:
```suggestion
TIP: See the guide for https://guides.micronaut.io/latest/micronaut-openapi-swagger-ui.html[Visualize with Swagger UI an OpenAPI specification of your Micronaut application.] to learn more.
```

**Review by sdelamo — APPROVED** [review]

**Pull request merged**